glue_jupyter/bqplot/common/tools.py

```diff
@@ -166,9 +170,6 @@ def update_from_roi(self, roi):
                 self.interact.selected_y = [np.min(roi.vy), np.max(roi.vy)]
             else:
                 raise TypeError(f'Cannot initialize a BqplotRectangleMode from a {type(roi)}')
-            # FIXME: the brush selector does not actually update unless the
-            # widget is resized/refreshed, see
-            # https://github.com/bloomberg/bqplot/issues/1067
```

Comment:
Not sure if this can go – I am still, though inconsistently, seeing the rectangle position not updating properly (right now of all things it does not seem to move at all in the Imviz viewer!). No idea if this is related to the original issue, but I'd rather leave the reminder in for now.

Reply:
Weird, it works for me. Huh. I'll add it back.